client/modules/IDE/components/SketchList.jsx

```diff
@@ -20,6 +21,15 @@ class SketchList extends React.Component {
     document.getElementById('sketchlist').focus();
   }
 
+  componentDidUpdate() {
+    if (this.props.sketches[0] === 'user not found') {
```

Comment:
This works, but not conventional IMO. I would prefer to render a 404 page over this (it can be fugly for now and we can design a nice one.)